docs/source/distributed._dist2.md

```diff
@@ -1,3 +1,5 @@
+# Experimental Object Oriented Distributed API
+
```

Comment:
Does this render duplicate h1s since there's already a title in the python file? 

We should probably remove it from there

Reply:
Removed - see below